### DISCUSSION TITLE
v0.22.1 on windows 10 can not close or minimize freetube client

### DISCUSSION THREAD

**Discussion by Spagha** [FreeTubeApp/FreeTube#6383] [Q&A] [answered]
After updating to v0.22.1 it isn't possible to minimize or close freetube. Taskbar is also hidden in freetube only. I have to use keyboard to open taskbar or use alt tab to close freetube or to switch to another task as e.g. to firefox. I guess it's a bug. I can't finde any information about that. This is in freetube only. Thank for help!

**Comment by zhfj1124** [comment]
From 0.22.0 to 0.22.1, some youtube videos cannot be played
 For example:https://youtu.be/oFz7tKjy-Ys
 This link is below https://www.youtube.com/channel/UCa6ERCDt3GzkvLye32ar89w
 All videos cannot be played, and they crash when they are played. Version 21.13 can be played.

**Reply by Spagha** [reply]
That is not the my problem. Client is not crashed. There is no problem to play videos. I am not able to close the task because there is no X in the right corner. And there is no taskbar in windows. In every other Application there is no problem with that. To close freetube I have to use taskmanager or other workarounds.

**Comment by kommunarr** [comment, marked as answer]
It sounds like your app is in fullscreen. [You can toggle this using the <kbd>F11</kbd> key.](https://docs.freetubeapp.io/usage/keyboard-shortcuts/)

**Reply by Spagha** [reply]
Thanks a lot. That helped!
